debian/changelog

```diff
@@ -0,0 +1,5 @@
+pim6sd (1.0.0-1) stable; urgency=medium
```

Comment:
The planned next version is 2.1.0. Other than that, this PR looks great!

Reply:
Thanks for the hint! I have adjusted that.